python/lsst/ctrl/execute/allocatorParser.py

```diff
@@ -148,6 +148,14 @@ def parseArgs(self, basename):
             default=None,
             help="glide-in inactivity shutdown time in seconds",
         )
+        parser.add_argument(
+            "--openfiles",
+            action="store",
+            dest="openfiles",
+            type=int,
+            default=20480,
```

Comment:
It would be nice to have this as a constant so we would only have to change it in one place.  Maybe a future ticket for changing default values to constants.